.travis.yml

```diff
@@ -32,10 +44,20 @@ matrix:
 #    - rvm: rbx-2
 #    - rvm: rbx-3
     - rvm: ruby-head
-    - rvm: 2.3
-      os: osx
+    - rvm: 2.6
+      env:
+        - RUBYOPT=--jit
 #    - rvm: jruby-1.7
 #    - rvm: jruby-9
   include:
-    - rvm: 2.3
+    - rvm: 2.6.0
+      os: osx
+    - rvm: 2.5.3
+      os: osx
+    - rvm: 2.4.5
+      os: osx
+    - rvm: 2.3.8
```

Comment:
I think it's too many versions on OS X - they're super slow to run anyways. Ideally the only version would be the system-Ruby on OS X, but I wasn't ever able to get that working in the Travis CI images.